setup.py

```diff
@@ -38,7 +38,7 @@
         'django-bootstrap-datepicker-plus',
         'django-crispy-forms',
         'django-extensions',
-        'django-import-export',
+        'django-import-export>=3.0.0b0',
```

Comment:
The feature to export one data model in different ways is only available in 3.x.

https://github.com/django-import-export/django-import-export/pull/1223